datasource/index.js

```diff
@@ -278,14 +287,15 @@ module.exports = yeoman.Base.extend({
         var moduleVersion = npmModule.split('@');
         var dependency = {};
         dependency[moduleVersion[0]] = moduleVersion[1];
+        debug('add dependency %s', util.inspect(dependency, {depth: null}));
         jsonfileUpdater(path.join(this.projectDir, 'package.json')).append(
           'dependencies', dependency, done
         );
       }
     }.bind(this));
-  },
+  };
```

Comment:
`;` isn't needed.

Reply:
I am surprised that `eslint` and/or `prettier` did not catch these extra `;`. Are we missing an eslint rule in our config?

Reply:
@bajtos see comment https://github.com/strongloop/generator-loopback/pull/363#issuecomment-416374651